client/header/activity-panel/panels/inbox/index.js

```diff
@@ -30,6 +31,21 @@ class InboxPanel extends Component {
 		this.props.updateCurrentUserData( userDataFields );
 	}
 
+	getUnreadNotesCount() {
```

Comment:
This could be unit tested, there's a fair bit of logic happening.

Reply:
Would also love to see some unit tests around this.

Reply:
Good idea! I added some tests for this functionality in the commit [6a00814](https://github.com/woocommerce/woocommerce-admin/pull/4386/commits/6a00814c19ea855976ef22acf1fe3617329511b0) in the [PR 4386](https://github.com/woocommerce/woocommerce-admin/pull/4386).